internal/app/mmlogic/mmlogic_service.go

```diff
@@ -88,11 +88,11 @@ func getPageSize(cfg config.View) int {
 		maxPageSize int = 10000
 	)
 
-	if !cfg.IsSet(name) {
+	if !cfg.IsSet(internal.StatestorePageSize) {
```

Comment:
One idea is to abbreviate this to just store? I ve seen this in previous teams - it communicates the intent - and is short. Thoughts?